baselines/pfsp/lib/Pool_ext.h

```diff
@@ -0,0 +1,57 @@
+#ifndef POOL_EXT_H
+#define POOL_EXT_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "PFSP_node.h"
+#include "c_bound_simple.h"
```

Comment:
It does not depend from B&B operators. Is it mostly because there is a MIN macro defined inside c_bound_simple used in some function of Pool_ext that if you remove the call to this library it will give a warning saying it is implicitly defined. I redefined macro MIN inside this file now. It should all be fine.

Reply:
Ok I got it. It's generally better to not mix/link all the libraries together.